src/lib.rs

```diff
@@ -500,7 +500,9 @@ mod tests {
         let result = compute_frame_ssimulacra2(source_data, distorted_data).unwrap();
         let expected = 8.764_571_f64;
         assert!(
-            (result - expected).abs() < 0.01f64,
+            // SOMETHING is WEIRD with Github CI where it gives different results across DIFFERENT
```

Comment:
Try comparing these two:
`RUSTFLAGS='-C target-cpu=x86-64-v2' cargo test -r`
and
`RUSTFLAGS='-C target-cpu=x86-64-v3' cargo test -r`

I think this is FMA again... v3 includes FMA, v2 doesn't. If the runners use different hardware (or are using a Rust toolchain with a different default target-cpu) then this is likely the problem

Reply:
Seems likely... And to be honest there's not a whole bunch I can do about it, given I can't control the runners. Other than loosen the parameters.